Flaky test: TestVolumeCreateClusterOpts

Looks like we have a new flaky test; I'm guessing something is using a map, and therefore randomized by Go. Introduced in https://github.com/docker/cli/pull/3606

```
--- FAIL: TestVolumeCreateClusterOpts (0.00s)
    create_test.go:195: assertion failed: 
        --- body
        +++ expectedBody
          volume.CreateOptions{
          	ClusterVolumeSpec: &volume.ClusterVolumeSpec{
          		... // 2 identical fields
          		AccessibilityRequirements: &{Requisite: {{Segments: {"region": "R1", "zone": "Z1"}}, {Segments: {"region": "R1", "zone": "Z2"}}, {Segments: {"region": "R1", "zone": "Z3"}}}, Preferred: {{Segments: {"region": "R1", "zone": "Z2"}}, {Segments: {"region": "R1", "zone": "Z3"}}}},
          		CapacityRange:             &{RequiredBytes: 1234, LimitBytes: 567890},
          		Secrets: []volume.Secret{
        - 			{Key: "key2", Secret: "secret2"},
          			{Key: "key1", Secret: "secret1"},
        + 			{Key: "key2", Secret: "secret2"},
          		},
          		Availability: "active",
          	},
          	Driver:     "csi",
          	DriverOpts: {},
          	... // 2 identical fields
          }
```


/cc @dperny

--- a/cli/command/volume/create.go
+++ b/cli/command/volume/create.go
@@ -3,6 +3,7 @@ package volume
 import (
 	"context"
 	"fmt"
+	"sort"
 	"strings"
 
 	"github.com/docker/cli/cli"
@@ -153,6 +154,9 @@ func runCreate(dockerCli command.Cli, options createOptions) error {
 				},
 			)
 		}
+		sort.SliceStable(volOpts.ClusterVolumeSpec.Secrets, func(i, j int) bool {
+			return volOpts.ClusterVolumeSpec.Secrets[i].Key < volOpts.ClusterVolumeSpec.Secrets[j].Key
+		})
 
 		// TODO(dperny): ignore if no topology specified
 		topology := &volume.TopologyRequirement{}

--- a/cli/command/volume/create_test.go
+++ b/cli/command/volume/create_test.go
@@ -3,6 +3,7 @@ package volume
 import (
 	"io"
 	"reflect"
+	"sort"
 	"strings"
 	"testing"
 
@@ -192,6 +193,9 @@ func TestVolumeCreateClusterOpts(t *testing.T) {
 
 	cli := test.NewFakeCli(&fakeClient{
 		volumeCreateFunc: func(body volume.CreateOptions) (volume.Volume, error) {
+			sort.SliceStable(body.ClusterVolumeSpec.Secrets, func(i, j int) bool {
+				return body.ClusterVolumeSpec.Secrets[i].Key < body.ClusterVolumeSpec.Secrets[j].Key
+			})
 			assert.DeepEqual(t, body, expectedBody)
 			return volume.Volume{}, nil
 		},

